examples/07-Train-an-xgboost-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>nit: Merlin Models API</p><p> </p><p>Maybe "A key feature of the Merlin Models API is tagging."</p><p> </p><p>s/can be/is/ and avoid parens again:</p><p> </p><p>"You can tag your data once, during preprocessing, and this information is picked up during later steps such as additional preprocessing steps, training your model, serving the model, and so on."</p><p> </p><p>I think I'd even use present tense in the last sentence--though clearly up to you:</p><p> </p><p>"During preprocessing that is performed by the <code>get_movielens</code> function, two columns in the dataset are assigned the <code>Tags.TARGET</code> tag:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>